src/rdkafka_partition.c

```diff
@@ -239,6 +254,7 @@ rd_kafka_toppar_t *rd_kafka_toppar_new0 (rd_kafka_topic_t *rkt,
         rktp->rktp_ops->rkq_opaque = rktp;
         rd_atomic32_init(&rktp->rktp_version, 1);
 	rktp->rktp_op_version = rd_atomic32_get(&rktp->rktp_version);
+        rd_kafka_toppar_op_version_bump(rktp->rktp_fetchq, rktp->rktp_op_version);
```

Comment:
I don't think there's any need for an OP_BARRIER for a newly created rktp, there is nothing to barrier.